src/main/java/org/eclipse/xpanse/terraform/boot/terraform/service/TerraformScriptsService.java

```diff
@@ -82,8 +83,19 @@ public TerraformResult destroyWithScripts(TerraformDestroyWithScriptsRequest req
     @Async("taskExecutor")
     public void asyncDeployWithScripts(
             TerraformAsyncDeployFromDirectoryRequest asyncDeployRequest) {
-        TerraformResult result = deployWithScripts(asyncDeployRequest);
-        log.info("Deployment service complete, {}", result.getCommandStdOutput());
+        TerraformResult result;
+        try {
+            result = deployWithScripts(asyncDeployRequest);
+            log.info("Deployment service complete, {}", result.getCommandStdOutput());
+        } catch (TerraformExecutorException e) {
```

Comment:
Catch all RuntimeExceptions and also remove throwing exceptions when SytemCmd is not successful. Instead return TerraformResult for both successful and failure cases.